go.mod

```diff
@@ -4,104 +4,113 @@ go 1.23.0
 
 require (
 	github.com/anchore/go-logger v0.0.0-20230725134548-c21dafa1ec5a
-	github.com/anchore/stereoscope v0.0.2-0.20240208195325-681f6715b0e3
-	github.com/anchore/syft v0.105.0
+	github.com/anchore/stereoscope v0.0.9
```

Comment:
pushed extra commit to fix misaligned dependencies that don't match expected vendoring of syft v1.17.0 like this one: https://github.com/anchore/syft/blob/v1.17.0/go.mod#L20